model/task/db.go

```diff
@@ -1785,7 +1707,39 @@ func FindActivatedStepbackTasks(projectId string) ([]Task, error) {
 		StatusKey:      bson.M{"$in": evergreen.TaskUncompletedStatuses},
 	})
 	if err != nil {
-
+		return nil, err
 	}
 	return tasks, nil
 }
+
+// FindActivatedStepbackTaskByName queries for running/scheduled stepback tasks with
+// matching build variant and task name.
+func FindActivatedStepbackTaskByName(projectId string, variantName string, taskName string) (*Task, error) {
+	t, err := FindOne(db.Query(bson.M{
```

Comment:
Can this be simplified down to `return FindOne(...`?

Reply:
No because FindOne takes in a query; that being said this is the DB file so the lack of abstraction is I think more okay here